torchao/experimental/kernels/cpu/aarch64/tests/CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(googletest)
 
+if (ANDROID_ABI)
+  # We are cross compiling, delay test discovery till runtime
+  set(CMAKE_GTEST_DISCOVER_TESTS_DISCOVERY_MODE PRE_TEST)
```

Comment:
why?

Reply:
IIUC they run some logic POST_BUILD (default). but tests are not runnable on host, when host != target, so this moves that to runtime (on target).

Reply:
TIL